packages/service-registry-api/src/client.rs

```diff
@@ -38,6 +44,13 @@ impl From<QueryMsg> for Error {
                 service_name,
                 chain_name: _,
             } => Error::Service(service_name),
+            QueryMsg::ServiceParamsOverride {
+                service_name,
+                chain_name,
+            } => Error::ServiceParamsOverride {
+                service_name,
+                chain_name,
+            },
```

Comment:
I find this trait implementation kind of strange. Can we convert it into a regular function? It's so odd to see `msg.into()` to create an error